pandas/tests/io/formats/test_to_string.py

```diff
@@ -0,0 +1,6 @@
+def test_to_string_encoding(float_frame,):
```

Comment:
these are all test_to_format.py, pls move there

would take a followup to split up test_to_format into 2 parts though (e.g. move the to_string out to a separate file)